prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -75,6 +75,42 @@
 #define FI_IBV_RDM_UNPACK_SERVICE_WR(value)     \
         ((void*)(uintptr_t)(value & (~(FI_IBV_RDM_SERVICE_WR_MASK))))
 
+// Send/Recv counters control
+
+#define FI_IBV_RDM_TAGGED_INC_SEND_COUNTERS(_connection, _ep, _send_flags) \
+do {                                                                	\
+	(_connection)->sends_outgoing++;                                \
+	(_ep)->pend_send++;                                             \
+	(_ep)->total_outgoing_send++;                                   \
+	(_send_flags) |= IBV_SEND_SIGNALED;                             \
```

Comment:
I'm agree with naming but decreasing occurs by handling of local completion request, so signaled flag is mandatory. I added to my todo list to rename this macro like "INC_SIG_SEND_COUNTERS"